src/lib/asn1/pss_params.h

```diff
@@ -13,7 +13,7 @@
 
 namespace Botan {
 
-class PSS_Params final : public ASN1_Object {
+class BOTAN_PUBLIC_API(2, 0) PSS_Params final : public ASN1_Object {
```

Comment:
This should be `3, 5` since 3.5 would be the first version where this was part of the public API

Reply:
3, 6 now

Reply:
Sorry for late answer forgot this since I have had a lot other things to do in my project.
Will change my path and test it ASAP. 3,6 is fine. I did not know that this was the version number before.

Thank you very much for helping me with my special case. It will be much easier for me now not having to patch when you got a new release.

Reply:
version number changed now and I rebased to the master branch.